src/script_handlers.c

```diff
@@ -306,6 +310,8 @@ void init_script_handlers() {
     // script_handlers[0x63].handler = handler_Stop;
     
     script_handlers[0x64].name = "CleanScreen";
+    
+    script_handlers[0x65].name = "PaintScreen";
```

Comment:
That's not a very informative name IMO. What does this command actually do?

Reply:
LoadBackground doesn't paint the background immediately.
OP_0x65() fades to black and then paints the background indicated in LoadBackground.
LoadEffect doesn't always need OP_65 to display, but if a fade to black is required you can use the OP_0x65() or FadeToBlack()

Reply:
So if I have

```
LoadBackground(BackgroundIdx);
LoadEffect(...);
OP_0x65();
```

will it draw the background identified with BackgroundIdx using the specified LoadEffect?

I think `DrawBackground` is a better name

Reply:
> So if I have
> 
> ```
> LoadBackground(BackgroundIdx);
> LoadEffect(...);
> OP_0x65();
> ```
> 
> will it draw the background identified with BackgroundIdx using the specified LoadEffect?

Yes, that would be the behavior.

> I think `DrawBackground` is a better name

I agree.
I think DrawBackground describes it best

Reply:
OK.
I have changed the command in script_handlers.c and edited the scripts (EN, JA, RU) with the respective changes.